src/content/popup/popup.ts

```diff
@@ -492,3 +498,91 @@ function getPopupArrow(): HTMLElement | null {
     ? hostElem.shadowRoot.querySelector('.arrow')
     : null;
 }
+
```

Comment:
Is there a better place for the below? Deferring to you @birtles here. `onHorizontalSwipe` could maybe go in its own file under `utils/` or similar? The next/prev functions I think are probably fine here, unless you want to move them into `tabs.ts`.

Reply:
Yeah, perhaps `src/content/popup/swipe.ts`?